src/i18n/locales/de-DE.json

```diff
@@ -104,6 +104,7 @@
   "cmps.designer.link.Channel.capacity": "Kapazität",
   "cmps.designer.link.Channel.sourceBalance": "Quellensaldo",
   "cmps.designer.link.Channel.destinationBalance": "Zielguthaben",
+  "cmps.designer.link.Channel.isPrivate": "Is private",
```

Comment:
Not sure what to do with non-English locales.

Reply:
You only need to update the `en-US.json` file. All of the other language files will be re-generated and overwritten by Crowdin.

Also, I'd prefer to capitalize the `P` to match the rest of the title case labels displayed in the sidebar.

Reply:
Updated capitalization here: 05b62b2a35c15538a0831ca2398b07a2c1de6d30